jbmc/src/janalyzer/CMakeLists.txt

```diff
@@ -24,3 +24,7 @@ target_link_libraries(janalyzer-lib
 # Executable
 add_executable(janalyzer janalyzer_main.cpp)
 target_link_libraries(janalyzer janalyzer-lib)
+
+target_compile_definitions(janalyzer-lib
+  PRIVATE "-DCBMC_VERSION=\"${CBMC_RELEASE} (${GIT_INFO})\""
```

Comment:
Rather than setting this on a per target basis in all the `CMakeLists.txt` files, could it be set once as additional CMAKE_CXX_FLAGS values in the top level `CMakeLists.txt` ?